View3D/Services/WsModelGeneratorService.cs

```diff
@@ -78,6 +81,69 @@ public void GenerateWsModel()
             }
         }
 
+        public void GenerateWsModelForWh2()
```

Comment:
This function should be pretty much the same for wh2 and wh3. Need to keep code duplication low